src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4751,7 +4751,11 @@ internal static string GetGetVerbText()
 param(
     [Parameter(ValueFromPipeline=$true)]
     [string[]]
-    $verb = '*'
+    $Verb = '*',
+
+    [Parameter(ValueFromPipeline=$true)]
+    [string[]]
+    $Group
```

Comment:
@iSazonov  What do you think would be the best way to accomplish this? Currently the groups are determined by looking at the class names and removing the word verb.

`[System.Reflection.IntrospectionExtensions]::GetTypeInfo([PSObject]).Assembly.ExportedTypes |
        Where-Object {$_.Name -match '^Verbs.'}`

I could put the current groups in a validate set but is that the best for long term support?